doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -238,7 +238,7 @@ nRF Desktop
 * Added support for :ref:`MCUboot <mcuboot_wrapper>` that allows to use the following features:
 
   * :ref:`Serial recovery DFU though USB <nrf_desktop_bootloader_serial_dfu>`
-  * :ref:`Background DFU <nrf_desktop_bootloader_background_dfu>` through :ref:`nrf_desktop_smp`
+  * :ref:`Background DFU <nrf_desktop_bootloader_background_dfu>` through :ref:`nrf_desktop_ble_smp`
```

Comment:
Do not modify old release notes.
I've fixed this and added the old tag on top of the new one.